app/serializers/review_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class ReviewSerializer < ActiveModel::Serializer
-  attributes :id, :user_screen_name, :formatted_date, :overall, :food, :price, :ambience, :service, :body
+  attributes :id, :user_screen_name, :formatted_date, :overall, :food, :price, :ambience, :service, :body, :total_votes
 end
```

Comment:
To pass `votes: Vote.by_current_user(user)` we can add the `:votes` attribute and then define it below like so, 

```
def votes
  Vote.by_current_user(scope)
end
```

Reply:
defined the helper method votes in the restaurant so that it's available in the restaurant level